openc3/lib/openc3/topics/command_decom_topic.rb

```diff
@@ -75,5 +75,35 @@ def self.get_cmd_item(target_name, packet_name, param_name, type: :WITH_UNITS, s
         end
       end
     end
+
+    def self.build_cmd(target_name, cmd_name, cmd_params, range_check, raw, scope:)
```

Comment:
This doesn't make sense in this file.  Maybe create a new DecomInterfaceTopic and move the other thing there too?